### DISCUSSION TITLE
(Alpha) v0.6.5

### DISCUSSION THREAD

**Discussion by AIRLegend** [AIRLegend/aitrack#116] [General]
# Improvements

- Changed "No active video feed" message as it was misleading for some users.
- Added configuration for disabling the tracking shortcut (#111, thanks @UNDEFINED-BEHAVIOR )
- Added initial steps for a 3D Head model scaling feature (Read more about it [here](https://github.com/AIRLegend/aitrack/wiki/About-3d-model-scale)).

# Solved bugs

- Improved frames per second steadiness (#93, thanks @Hugal31!)
- Fixed a bug which wouldn't allow the FOV value to be saved.

Keep in mind this version is an alpha and bugs/malfunctions could be present. In case you encounter any issue, please, post it  on the [issues section](https://github.com/AIRLegend/aitracker/issues) or join our discord server to report it.

<a href="https://discord.gg/3XHY4xSDGv"><img src="https://image.flaticon.com/icons/svg/2111/2111370.svg" width="30px"/></a> Join the server!

<hr /><em>This discussion was created from the release <a href='https://github.com/AIRLegend/aitrack/releases/tag/v0.6.5-alpha'>(Alpha) v0.6.5</a>.</em>

**Comment by busman137** [comment]
So I was so excited to use this programe.  Bought a new logitech c922 camera and activate aitrackv0.6.5 and get the spinning wheel.  Then the "No Camera Detected" message.  Then I clicked ok and got the next window message that said: "There was a problem initializing the tracker. Check the models folder and restart the program".  What ever that means !!!!  Super disappointed.   Any suggestions ?

**Reply by AIRLegend** [reply]
Hi

The first problem, seems like a drivers issue. You should check them as that model is definitely compatible. The second message suggest that you've placed `AITrack.exe` on a folder where it has no read/write permissions and/or there isn't next to the other files that are shipped in the release zip.

**Reply by searching46dof** [reply]
For the first message, busman137 can verify if the driver is installed correctly by using a 3rd party webcam app.
Problems initializing the tracker may be missing or inaccurate settings for FOV, distance, video resolution (width & height) and framerate.

**Comment by wyliu09** [comment]
idk why i cant open the app
after i clicked the app, it flashed few seconds and closed itself.
LOG:
[2022-07-26 17:35:11.451] [aitrack] [info]  ----------  AITRACK LOG   ----------
[2022-07-26 17:35:11.809] [aitrack] [info] Created/Found prefs.ini
[2022-07-26 17:35:11.809] [aitrack] [info] Searching for cameras...

**Reply by searching46dof** [reply]
You may want to verify that FOV, distance, camera video resolution (width & height) and frame rate are all set and that the values are accurate.

**Reply by AIRLegend** [reply]
@wyliu09 Seems to me that AITrack has trouble when searching/accessing the camera. Maybe it's a driver issue.

**Reply by searching46dof** [reply]
It's already been mentioned about a possible driver issue. That can be verified by using another app to access the camera.

I was pointing out another possibility since I encountered this when I first used AItrack without any settings and without reading any documentation. The sources indicates it requires the correct parameters otherwise it would cause an exception in the tracker. I also tried parameters unsupported by the driver (resolution & framerate) before obtaining the manufacturer info and it crashed the driver

**Comment by Versande** [comment]
Getting an error, when I double click the executable I get "AITrack.ece - System Error, the code execution cannot proceed because Qt5Widgets.dll was not found. Reinstalling the program may fix this problem" Reinstalled multiple times and he folder clearly has the qt5widget file.

**Reply by Versande** [reply]
All of a sudden it works now, lol.
